src/bootstrap-table.js

```diff
@@ -1080,7 +1080,8 @@ class BootstrapTable {
             const props = key.split('.')
 
             for (let i = 0; i < props.length; i++) {
-              if (value[props[i]] !== null) {
+              // eslint-disable-next-line eqeqeq
```

Comment:
An alternative to this is to update the ESLint config to use `"eqeqeq": ["error", "smart"]` instead of just `"eqeqeq": "error"`.

Reply:
This could also be replaced with the functionality of `getItemField` and consider any falsy value:

https://github.com/wenzhixin/bootstrap-table/blob/eab8d6f1bad7b2371c3560fd696592e7429a513f/src/utils/index.js#L492-L494